source/includes/_simulator-guide.html.md

```diff
@@ -0,0 +1,130 @@
+# Simulator Training
+
+> ![Simulator Diagram](../images/brain-sim-marketing.png)
+
+This guide is intended to give you information and best practices you’ll need to train a BRAIN on The Bonsai Platform with simulators. It covers best practices such as running simulations in parallel, general guidelines like how to use corporate proxies with the CLI, and other tips pertaining to simulators and simulations.
+
+Currently, the Bonsai Platform only uses simulators for applied deep reinforcement learning (DRL). For more background on applied DRL with simulations read our blog post on [simulations as a training environment][6].
+
+We have simple coordinators for Python & C++, two common languages used in systems modeling. We also have coordinators for Simulink, EnergyPlus and Gazebo. To learn more about connecting your simulator, refer to our [Library Reference][4].
+
+## What is a simulator?
+
+A simulator is a computational model of some external process governed by a field of mathematics. Robotics, industrial automation, supply chain logistics, and structural engineering are all domains which use simulators to test how simplified models of complex problems will behave.
+
+Deep Reinforcement Learning can work with simulated models to train a BRAIN to perform tasks within the modeled system. Tasks can be as simple as "stand this pole upright" to as complex as "learn to walk." 
+
+To be effective, a BRAIN needs to be trained using DRL against a simulated model. The Bonsai SDK allows the DRL system to control your simulator during the training process. Any simulator which has an initial state, and can be incremented through time in a stepwise fashion can be connected to the Bonsai training system.
+
+## Currently Integrated Simulators
+
+The Bonsai Platform currently supports simulators written in our Python or C++ libraries, or through coordinators (bridges/connections) of these languages. The following are also available as coordinators built in-house:
```

Comment:
_in_ these languages. Currently available coordinators:

Also, "coordinator" is the word we use internally, but maybe we can avoid defining that here and say "we have integrated the following sims" instead?